docs/SRS-Volere/SRS.tex

```diff
@@ -677,8 +677,13 @@ \section*{Appendix --- Reflection}
 
   \medskip
   \textbf{Jason Nam}\\
-  \textit{Chosen Skills}: \\
-  \lips
+  \textit{Chosen Skills: (a), (c), (g)}\\
```

Comment:
Your role was NLP and ML in the dev plan so I think you should add that  to the chosen skills. You can keep the other material but just add this to the chosen skills.